packages/css-data/bin/mdn-data.ts

```diff
@@ -17,6 +17,18 @@ import type {
 } from "@webstudio-is/css-engine";
 import * as customData from "../src/custom-data";
 
+/**
+ * Store prefixed properties without change
+ * and convert to camel case only unprefixed properties
+ * @todo stop converting to camel case and use hyphenated format
+ */
+const normalizePropertyName = (property: string) => {
+  if (property.startsWith("-")) {
```

Comment:
hmm, so you don't camel-case the name if it starts with dash? is it for vendor prefixes or is it for variables? I am guessing for vendors, but then you will return lower case name and in other cases camel case ... I don't get the logic

Reply:
Both custom properties and prefixed properties are not camel cased. Ideally convert all other properties to hyphenated as well.

Reply:
but then this function returns camel cased for everything else and hyphenated for variables and vendor prefixed once? this seems inconsistent and werid

Reply:
We wanted to move away from camel cased anyway. Why add more?

Reply:
by creating inconsistency in the meantime it will create more confusion, lets do the switch at once

Reply:
Custom property is inconsistent and should not be camel cased

Reply:
actually you are right regarding custom property, it could be inconsistently named by the user, so it can't be reliably converted to camel case and then restored back to hyphenated